multitenancy/tenant-manager-api/src/main/resources/application.properties

```diff
@@ -28,10 +28,10 @@
 
 %prod.quarkus.flyway.locations=db/migration/postgresql
 
-quarkus.hibernate-orm.database.default-schema=tenantmanager
+quarkus.hibernate-orm.database.default-schema=${TENANT_MANAGER_DEFAULT_SCHEMA:tenantmanager}
```

Comment:
@famartinrh How about this change so that we can move past this issue for now?  Although I'd still like to have a better solution to a problem that seems to be OS-specific.

Reply:
I'm fine with this, and yes absolutely, there is something very wrong behind this issue that I would also still like to fix better.

I think it may be just that hibernate makes all sql strings lowercase...

Reply:
FWIW there is an h2 setting that may be relevant - `DATABASE_TO_UPPER` - I'm unclear on the semantics but it sounds interesting.